### ISSUE TITLE
Loinc codes in testcase 7A (ID40) missing in Retam 

### ISSUE THREAD

**Issue by AlexisVZ-MDS** [hl7-be/lab#327]
Hello,

I  just noticed for example the code 1527-1, is "present" in the reference test file for testcase 7A, but not yet in retam (checked through https://www.vas.ehealth.fgov.be/webretam/retam/home.htm?eventName=SEARCH_BY_LOINC_CODE&htmlfield_searchloinc=1527-1 and in download [BE subset LOINC_2024_Q2](https://apps.health.belgium.be/terminology-portal-public-rs/documents/download/39) )

Same for 1507-3, 1496-9, 1518-0, 1552-9, 14796-7, 72902-0, 72901-2

![image](https://github.com/user-attachments/assets/a208211b-b1a0-4b2d-9551-5a245dcf7719)

**Comment by bdc-ehealth** [comment]
@AlexisVZ-MDS 

that is correct. Although ReTaM codes are preferred, because of the slow update rate of ReTaM, we also allow international LOINC codes. Please send a mail to terminologie@health.fgov.be and take a look a the FAQ for a complete request: https://github.com/hl7-be/lab#:~:text=What%20information%20do%20I%20need%20to%20provide%20when%20requesting%20a%20new%20ReTaM%20code%3F
